site/_docs/configuration.md

```diff
@@ -571,11 +571,11 @@ file or on the command-line.
 # Where things are
 source:       .
 destination:  ./_site
-plugins_dir:  ./_plugins
-layouts_dir:  ./_layouts
-data_dir:     ./_data
-includes_dir: ./_includes
-collections:  null
+plugins_dir:  _plugins
+layouts_dir:  _layouts
+data_dir:     _data
+includes_dir: _includes
+collections:  {}
```

Comment:
I see now. Is there a reason that the DEFAULTS hash does not actually contain that? As far as I can tell, it only forces `posts: output` to be true even if explicitly set to false. Seems odd for `posts` to be special in this way.